### ISSUE TITLE
Access to mobile version of cheatsheet

### ISSUE THREAD

**Issue by cmgordon** [ctpc/nucoach-issues#55]
Some participants have lost their cheat sheets. I will create a digital version using ionic creator. Perhaps this can be added as a new module similar to the Healthy Tips?

**Comment by imana97** [comment]
yes, and it can be on the module interface... accessible from profile... module.
send me the html and I can add it. easy.

**Comment by imana97** [comment]
Hi, do you have this html? If so, I can create the module.
@cmgordon, let me know
